appveyor-linux-binary.yml

```diff
@@ -214,32 +212,14 @@ for:
       - sh: "pytest -vv -n 4 tests/integration/buildcmd/test_build_terraform_applications.py --json-report --json-report-file=TEST_REPORT-integration-buildcmd.json"
       - sh: "pytest -vv -n 4 tests/integration/buildcmd/test_build_terraform_applications_other_cases.py --json-report --json-report-file=TEST_REPORT-integration-buildcmd.json"
 
-  # Integ testing deploy
+  # Integ testing package & delete
   -
     matrix:
       only:
-        - configuration: DeployIntegTesting
+        - configuration: PackageAndDeleteAndDeployIntegTesting
```

Comment:
We have too much duplication between the binary jobs and build-from-source jobs. We should look to abstract away some of this shared configuration